### ISSUE TITLE
linebreak-style rule misses linebreaks

### ISSUE THREAD

**Issue by weswigham** [palantir/tslint#1455]
### Bug Report
- **TSLint version**: `3.14.0`
- **TypeScript version**: `Version 2.1.0-dev.20160803`
- **Running TSLint via**: CLI / Node.js API
#### TypeScript code being linted

``` ts

/* @internal */
namespace ts {
    /**
     * Ternary values are defined such that
     * x & y is False if either x or y is False.
     * x & y is Maybe if either x or y is Maybe, but neither x or y is False.
     * x & y is True if both x and y are True.
     * x | y is False if both x and y are False.
     * x | y is Maybe if either x or y is Maybe, but neither x or y is True.
     * x | y is True if either x or y is True.
     */
    export const enum Ternary {
        False = 0,
        Maybe = 1,
        True = -1
    }
}
```

(Github normalized line endings, but the line with `x & y is False if either x or y is False` should only end in `LF`)

with `tslint.json` configuration:

``` json
{
    "rules": {
        "linebreak-style": [true, "CRLF"]
    }
}
```
#### Actual behavior

Reports no error.
#### Expected behavior

Error on line `x & y is False if either x or y is False` because the line only ends with `LF`.

This rule instantiates and uses a scanner from `typescript` for finding all the tokens in the file - as it turns out, the scanner in TS skips recording a lot of newline tokens when they're unnecessary whitespace/in a comment/etc.

**Comment by JKillian** [comment]
cc @patsissons 

Maybe this rule can be switched to just parse `sourceFile.getFullText()` by hand without using `ts.Scanner`?

**Comment by ajafff** [comment]
Fixed by #2394
